app/services/evidence/rules/20240426042117_national_savings_account.rb

```diff
@@ -6,18 +6,12 @@ class NationalSavingsAccount < Rule
       key :capital_nsa_19
       group :capital
 
```

Comment:
@arthurashman this rule is a good example of the improvements from using your suggestion of using submitted application for evidence prompt rules. Less code duplication, less complexity 🏆